### ISSUE TITLE
How to convert text to target audio（TTS） using ns3_codec（naturalspeech3）

### ISSUE THREAD

**Issue by aguang1201** [open-mmlab/Amphion#166]
Thanks for the excellent work, noticed that ns3_codec can blend two audio perfectly together, but if I have a text and an audio, how do I get the audio to read out the text content with ns3_codec?

**Comment by HeCheng0625** [comment]
Hi, since FACodec is not a ASR model, it can only get content representation from content codes, you can check https://github.com/open-mmlab/Amphion/blob/main/models/codec/ns3_codec/README.md

**Comment by aguang1201** [comment]
Thanks for the reply and sorry I wasn't very clear before. I mean as a text-to-speech (TTS) model, it should support inputting a piece of text instead of speech, is it that FACodec is not a TTS model and it only supports speech input? Is it possible that the NaturalSpeech 3 will do what I want?

**Comment by aguang1201** [comment]
@HeCheng0625 Thanks, I seem to get it, FACodec is not a TTS model, please let me rephrase my question, how does FACodec work with a TTS model?

**Comment by HeCheng0625** [comment]
Hi, you can follow the FACodec and NaturalSpeech 3 paper: https://arxiv.org/abs/2403.03100; we are working on reproducing NS3.

**Comment by LearnMF** [comment]
How to use naturalspeech3 for tts ; Is there any demo python code ?

**Comment by HeCheng0625** [comment]
Hi, currently NS3 is not open source due to commercial reasons, and we are still working hard to reproduce it.

**Comment by yuantuo666** [comment]
Hi @aguang1201, if you have any further questions, feel free to re-open this issue. We are glad to follow up!
